Tests/SuperLinq.Test/RankTest.cs

```diff
@@ -124,28 +204,53 @@ public sealed record Person(string Name, int Age, int ExpectedRank);
 				new Person(Name: "Jim", Age: 74, ExpectedRank: 8),
 				new Person(Name: "Jes", Age: 11, ExpectedRank: 1),
 		}
-			.GetTestingSequence()
+			.GetTestingSequence(maxEnumerations: 3)
+			.Select(x => new object[] { x, });
+
+	public static IEnumerable<object[]> GetPersonSequences2() =>
+		new[]
+		{
+				new Person(Name: "Bob", Age: 11, ExpectedRank: 1),
+				new Person(Name: "Sam", Age: 11, ExpectedRank: 1),
+				new Person(Name: "Kim", Age: 11, ExpectedRank: 1),
+				new Person(Name: "Tim", Age: 23, ExpectedRank: 4),
+				new Person(Name: "Joe", Age: 23, ExpectedRank: 4),
+				new Person(Name: "Mel", Age: 28, ExpectedRank: 6),
+				new Person(Name: "Jim", Age: 28, ExpectedRank: 6),
+				new Person(Name: "Jes", Age: 30, ExpectedRank: 8),
```

Comment:
I'd like to see these items in a non-sequential order, such as 4, 1, 6, 8, etc... This would be to confirm that equal rankings work properly regardless of the input data order.